assets/js/data/utils/update-state.ts

```diff
@@ -1,33 +1,37 @@
 /**
- * External dependencies
+ * Utility for updating nested state in the path that changed.
  */
-import { klona } from 'klona/json';
+function updateNested< Type >( // The state being updated
```

Comment:
I usually saw this done using just `T` as the generic name - @gigitux what do you think of this?

Reply:
Yes, `T` is the standard for the generic type

Reply:
Ok I'll change it. I struggle to read and write generics when they use shorthand instead of `Type` 🙈